### DISCUSSION TITLE
view same mindmap multiple times within a FP instance

### DISCUSSION THREAD

**Discussion by nnako** [freeplane/freeplane#464] [Help] [answered]
I was wondering if it was possible to view the same mindmap **multiple times** within a single Freeplane instance. There would be at least two use cases where this would make great sense:

![image](https://user-images.githubusercontent.com/5845602/175106602-d6e38f7c-dbd4-49a9-9084-0bce2a6115ea.png)

- when I use tabs as **project tabs**, where each main tab (on the main tab row below the tool row) is used for group mindmaps respective to the relevance for a particular "project". Here, I might use a specific mindmap which I have already used on another project tab. So, I would't have to switch main tabs to find a particular mindmap which I am currently using within another project (on another main tab).

- when I use a rather **big mindmap**, I find myself scrolling around. Just to end up frustrated as I have lost focus while searching the big mindmap. I would rather be able to display that big mindmap on two different secondary tabs within the same main tab. In each secondary tab, I could scroll the big map so that I view a different part of the same mindmap.

Of course, this would require the ability to load a mindmap into another "view" even though it already exists somewhere else. And, the ability to synchronize changes made to the same mindmap within different views. And, surely, a lot of other technical challenges...

Would this be possible? Would you people be interested in such a feature?

**Comment by Illegit** [comment, marked as answer]
Interesting point
I do not understand your notes because I imagine you utilize the command "new map view". therefore if you already know that what is your point?
I'd like if you highlight what I'm missing because I normally use that command, but sometimes working I got confusing, and I like to have suggestion on how to improve my usage.
Thank you
Gitille

**Reply by nnako** [reply]
Hi @Illegit ,

thanks for your quick reply.

Indeed, the feature you mentioned already solves my request. I was searching the discussions for a solution, before, but couldn't find any hint. So, there is no need for highlighting anything further ;-) . I even see that `new map view` is already an old feature, implemented before v1.3.15 .

**Reply by Illegit** [reply]
OK

**Comment by nnako** [comment]
Concerning viewing the same mindmap within different panes in one main tab, there seems to be one effect, though, which is kind of annoying:

- even though the current node selection doesn't change in the other pane, when I work on one of them,
- the mindmap gets vertically shifted within the other pane when I open / close trees on one of them

this means, while working on one of the two views of the same mindmap, the focus within the other mindmap will not stay where it is, but gets lost. yes, when clicking into the other mindmap, the selection returns into persoective, but most probably the selection will be located at the top or bottom pane border.

Is there a way to prevent the other focuses from moving when working in one mindmap view?

**Reply by dpolivaev** [reply]
I created a new issue from this comment but I can not see the problem. Could you please add some evidence, attach the mind map where you have this problem and describe how to reproduce it?

Please comment at https://github.com/freeplane/freeplane/issues/466

**Reply by dpolivaev** [reply]
But please check how it works in version 1.10.3 released today. Actually, I worked on map scrolling, and this issue might be fixed.

**Comment by nnako** [comment]
I just checked the feature to "**view maps multiple times**" within `Freeplane v1.10.05-pre04` and it looks very stable to me:

- visible region for each view is independent from all other views
- zooming within one view does not affect the other view
- even folding / unfolding of trees only occurs within the selected view, only

unfortunately, though, every 2nd, 3rd, ... view of a mindmap is not re-loaded into its last location within the open tabs after closing (and saving) the mindmaps and re-opening Freeplane. After re-open of Freeplane, only the original views get recovered. So, the layout of maps within the tabs have to be re-done each time Freeplane starts.

**Would there be a way to save the existence of further views as well as their locations within the Freeplane tabs?**

**Reply by euu2021** [reply]
see #512 for a related discussion. It would be cool to have a script that uses the Presentations to reopen all the views with only one click.
